### PR TITLE
Feature/sep6 35 create actor page

### PR DESCRIPTION
**Pull request by Tabita-Roxana**

### CODE REVIEW

File: backend/src/main/java/com/sep6/backend/controller/ActorsController.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 @RestController
 @RequestMapping("/actors")
 @AllArgsConstructor
+@CrossOrigin(origins = "http://localhost:3000")
```
Comment (GeorgianaIon) [inline backend/src/main/java/com/sep6/backend/controller/ActorsController.java:18]: Delete this

File: backend/src/main/java/com/sep6/backend/controller/AuthenticationController.java [hunk 2]
```diff
@@ -7,21 +7,15 @@
...
-import org.springframework.web.server.ResponseStatusException;
+import org.springframework.web.bind.annotation.*;
 
 import java.io.IOException;
-import java.util.NoSuchElementException;
 
 @RestController
 @RequestMapping("/auth")
 @RequiredArgsConstructor
+@CrossOrigin(origins = "http://localhost:3000")
```
Comment (GeorgianaIon) [inline backend/src/main/java/com/sep6/backend/controller/AuthenticationController.java:18]: And this

File: frontend/package.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.4.0",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (GeorgianaIon) [inline frontend/package.json:15]: Uninstall babel

File: frontend/src/AppRouter.tsx [hunk 4]
```diff
@@ -20,6 +21,7 @@ const AppRouter: React.FC = () => {
             <Route path="/"  element={<HomePage/>} />
             <Route path="/sign-up"  element={<SignUp/>} />
             <Route path="/login"  element={<Login/>} />
+            <Route path="/actor"  element={<PeoplePage/>} />
```
Comment (GeorgianaIon) [inline frontend/src/AppRouter.tsx:24]: maybe the path should look different

File: frontend/src/Pages/people.tsx [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+  const [people, setPeople] = useState<People>();
+
+
+  const baseUrl = 'https://image.tmdb.org/t/p/original'
+
+
+  useEffect(() => {
+    const fetchActor = async () => {
+      try {
+        const people= await actorService.getSpecificActor(3136)
```
Comment (GeorgianaIon) [inline frontend/src/Pages/people.tsx:43]: the id is hardcoded, maybe you can take the id from the url query and use that

**Review by GeorgianaIon — CHANGES_REQUESTED** [review]

**Review by KhaledHammoun — APPROVED** [review]

**Pull request closed without merging**